### PR TITLE
Feature/mx 2114 seed script

### PR DESCRIPTION
**Pull request by vyvytranngoc**
### PR Context
 seed script to fill the backend / database with artificial data. implemented as an entry-point in mex-backend (in the pyproject.toml) and accepts parameters to configure count, chattiness, seed

### Added

- tests\seed\test_main.py and integrations test that runs the entrypoint with a small count (eg 25) and checks the items end up in neo4jv

### CODE REVIEW

File: mex/backend/seed/main.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+@click.option("--seed", default=1, type=int)
+@click.option("--chattiness", default=8, type=int)
+@click.option("--locale", default="de_DE", type=str)
+def main(
+    count: int,
+    seed: int,
+    chattiness: int,
+    locale: str,
+) -> None:
+    """Entrypoint for seed script."""
```
Comment (esinsj) [inline mex/backend/seed/main.py]: in the docstring, could you provide a more telling description of what the function does and how to use the entry point and what the parameters do?
like

""" fill the backend / database with artificial data

can be invoked via entry point, and pass optional parameters like:
uv run backend-seed --count 50 --locale en_US


Args:
  count: approximate number of items to be created. Default: 25
  ...
"""

File: tests/seed/test_main.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    graph = get_graph()
+
+    graph_identifiers = {node["identifier"] for node in graph if node.get("identifier")}
+
+    expected_identifiers = {
+        str(item.identifier) for item in expected_items if hasattr(item, "identifier")
+    }
+
+    assert expected_identifiers <= graph_identifiers
```
Comment (esinsj) [inline tests/seed/test_main.py]: Maybe add a comment why expected_identifiers will never be more than graph_identifiers.
Also, all expected_identifiers should be contained in graph_identifiers, or? should that be tested? up for discussion
Maybe also add a test that graph_identifiers are more 0 (maybe even 5 as minimum is possible)

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- seed script to fill the backend / database with artificial data: tests\seed\test_main.py and integrations test: tests\seed\test_main.py
```
Comment (esinsj) [inline CHANGELOG.md]: I think you can leave out the new file names here, but you could add that the entrypoint can be invoked with "uv run backend-seed"

**Review by esinsj — COMMENTED** [review]
Nice work! 
Just some small ideas about documentation and testing :)

File: tests/seed/test_main.py (hunk 2 above)
Comment (vyvytranngoc) [reply tests/seed/test_main.py]: <= checks that **all** identifiers generated by the seed script are present in the database after running the entrypoint :)

Comment (esinsj) [reply tests/seed/test_main.py]: Yes, you're absolutely right! I misread the code here, and your test already covers this. My mistake 😅
I still would like to add the comment though? Something like

```suggestion
   # graph_identifiers contains all IDs of expected_identifiers + IDs of merged items
    assert expected_identifiers <= graph_identifiers 
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- seed script to fill the backend / database with artificial data
+- entrypoint to be invoked with "uv run backend-seed"
```
Comment (esinsj) [inline CHANGELOG.md]: with the dash for the entrypoint explanation it isn't so abvious, that the previous bulletpoint is meant.

```suggestion
- seed script to fill the backend / database with artificial data. entrypoint can be
    invoked with "uv run backend-seed"
```

**Review by esinsj — DISMISSED** [review]
Nice, just two small suggestions for documentation, but no show stopper.

**Review by esinsj — APPROVED** [review]
